src/main/java/org/cloudbees/literate/jenkins/LiterateMultibranchProject.java

```diff
@@ -310,7 +304,11 @@ public TopLevelItem newInstance(ItemGroup parent, String name) {
          */
         @SuppressWarnings("unused") // stapler
         public BranchProjectFactoryDescriptor getProjectFactoryDescriptor() {
-            return Jenkins.getInstance().getDescriptorByType(ProjectFactoryImpl.DescriptorImpl.class);
+            Jenkins j = Jenkins.getInstance();
+            if (j == null) {
+                throw new IllegalStateException(); // TODO 1.590+ getActiveInstance
```

Comment:
please add a message - if a user ever sees this having ("Jenkins is starting up or shutting down") is highly valuable.